hoomd/simulation.py

```diff
@@ -187,6 +215,12 @@ def create_state_from_gsd(self,
             automatically selected direction. The default value of ``(None,
             None, None)`` will automatically select the number of domains in all
             directions.
+
+        .. rubric:: Example:
+
+        .. code-block:: python
+
+            simulation.create_state_from_gsd(filename = path / 'file.gsd')
```

Comment:
I think that adding a variable `filename` to the invisible code block would be more understandable for the new Python user as `Path` objects are not well known from my estimation.

Reply:
I'm not certain that `filename=filename` would be any less confusing to novice users.

Reply:
I agree that `filename = filename` might not be the best choice as well. Maybe we can rename `path` to something more specific such as `path_to_input_file`.

Reply:
It isn't so much the name, but the `Path("/path/to/tmp/dir") / "file.gsd")` syntax that I thought might be advanced.